#! /bin/sh /usr/share/dpatch/dpatch-run
## 05_add_doctype_to_catalog_xml.dpatch by Adam Di Carlo <aph@debian.org> and
##     and Daniel Leidert <daniel.leidert@wgdd.de>.
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: * docbook-4.{2,3,4,5}/catalog.xml: Adds the missing DOCTYPE.
## DP:
## DP: <URL:http://packages.debian.org/changelogs/pool/main/d/docbook-xml/current/changelog#versionversion4.2-8>
## DP: <URL:http://packages.debian.org/changelogs/pool/main/d/docbook-xml/current/changelog#versionversion4.5-1>

@DPATCH@
diff -urNad docbook-xml-4.5~/docbook-4.2/catalog.xml docbook-xml-4.5/docbook-4.2/catalog.xml
--- docbook-xml-4.5~/docbook-4.2/catalog.xml	2007-03-25 22:16:26.000000000 +0200
+++ docbook-xml-4.5/docbook-4.2/catalog.xml	2007-03-25 22:17:56.000000000 +0200
@@ -1,4 +1,7 @@
 <?xml version='1.0'?>
+<!DOCTYPE catalog PUBLIC "-//OASIS//DTD Entity Resolution XML Catalog V1.0//EN"
+    "http://www.oasis-open.org/committees/entity/release/1.0/catalog.dtd">
+
 <catalog xmlns="urn:oasis:names:tc:entity:xmlns:xml:catalog">
 
 <!-- ...................................................................... -->
diff -urNad docbook-xml-4.5~/docbook-4.3/catalog.xml docbook-xml-4.5/docbook-4.3/catalog.xml
--- docbook-xml-4.5~/docbook-4.3/catalog.xml	2007-03-25 22:16:26.000000000 +0200
+++ docbook-xml-4.5/docbook-4.3/catalog.xml	2007-03-25 22:18:09.000000000 +0200
@@ -1,4 +1,7 @@
 <?xml version='1.0'?>
+<!DOCTYPE catalog PUBLIC "-//OASIS//DTD Entity Resolution XML Catalog V1.0//EN"
+    "http://www.oasis-open.org/committees/entity/release/1.0/catalog.dtd">
+
 <catalog xmlns="urn:oasis:names:tc:entity:xmlns:xml:catalog">
 
 <!-- ...................................................................... -->
diff -urNad docbook-xml-4.5~/docbook-4.4/catalog.xml docbook-xml-4.5/docbook-4.4/catalog.xml
--- docbook-xml-4.5~/docbook-4.4/catalog.xml	2007-03-25 22:16:26.000000000 +0200
+++ docbook-xml-4.5/docbook-4.4/catalog.xml	2007-03-25 22:18:24.000000000 +0200
@@ -1,4 +1,7 @@
 <?xml version='1.0'?>
+<!DOCTYPE catalog PUBLIC "-//OASIS//DTD Entity Resolution XML Catalog V1.0//EN"
+    "http://www.oasis-open.org/committees/entity/release/1.0/catalog.dtd">
+
 <catalog xmlns="urn:oasis:names:tc:entity:xmlns:xml:catalog" prefer="public">
 
 <!-- ...................................................................... -->
diff -urNad docbook-xml-4.5~/docbook-4.5/catalog.xml docbook-xml-4.5/docbook-4.5/catalog.xml
--- docbook-xml-4.5~/docbook-4.5/catalog.xml	2007-03-25 22:16:26.000000000 +0200
+++ docbook-xml-4.5/docbook-4.5/catalog.xml	2007-03-25 22:18:37.000000000 +0200
@@ -1,4 +1,7 @@
 <?xml version='1.0'?>
+<!DOCTYPE catalog PUBLIC "-//OASIS//DTD Entity Resolution XML Catalog V1.0//EN"
+    "http://www.oasis-open.org/committees/entity/release/1.0/catalog.dtd">
+
 <catalog xmlns="urn:oasis:names:tc:entity:xmlns:xml:catalog" prefer="public">
 
 <!-- ...................................................................... -->
